### ISSUE TITLE
[QUESTION] What does "maker" stand for?

### ISSUE THREAD

**Issue by mrgoonie** [binance-exchange/node-binance-api#70]
In websocket, there is "maker" boolean in response message, what does it stand for? Is it BUY & SELL? If so, "true" is BUY or SELL?

Thanks.

**Comment by jaggedsoft** [comment]
A maker is the first person to place the limit order

So let's say you place an order to buy or sell something at 0.002 btc
Whoever takes your order is the taker
Whoever posts the limit order is the maker

**Comment by mrgoonie** [comment]
I see, thank you for the information!

**Comment by jaggedsoft** [comment]
No problem. Have a wonderful day :)

**Comment by thechile** [comment]
but binance does not provide `side` right ? I mean none of the API responses give you `side`.

**Comment by jaggedsoft** [comment]
Not that I have seen yet
The closest I can find uses [candlestick data](https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md#klinecandlestick-data) to compare buy volume compared to total volume over a period (say 5 minutes, or 1 hour)
With this data you could say that there is 60% buying demand over a period

**Comment by thechile** [comment]
I took a look at binance webpage, for any coins market page on the right hand side it has the order book activity which has red and green for buy and sell orders. And this feed uses the same websocket data. 

It seems binance use the ismaker field of the ws data to signify the side. To me 'maker' is related to someone placing a limit order to add liquidity to the books but they seem to use it just to mean are they the seller or buyer via the true/false. Possible language/translation thing.

Maybe someone else can verify this is true/accurate ?  I used chrome dev tools to compare the feed frames with the green/red updates in the feed.
